include/seqan3/alphabet/quality/phred68.hpp

```diff
@@ -0,0 +1,293 @@
+     *
+     * \details
+     *
+     * Satisfies the seqan3::alphabet_concept::to_char() requirement via the seqan3::to_char() wrapper.
+     *
+     * \par Complexity
+     *
+     * Constant.
+     *
+     * \par Exceptions
+     *
+     * Guaranteed not to throw.
+     */
+    constexpr rank_type to_rank() const
+    {
+        return _value;
+    }
+    //!\}
+
+    /*!\name Write functions
+     * \{
+     */
+     /*!\brief Assign from a character.
+      *
+      * \details
+      *
+      * Satisfies the seqan3::alphabet_concept::assign_char() requirement via the seqan3::assign_char() wrapper.
+      *
+      * \par Complexity
+      *
+      * Constant.
+      *
+      * \par Exceptions
+      *
+      * Guaranteed not to throw.
+      */
+    constexpr phred68 & assign_char(char_type const c)
+    {
+        assert(c >= offset_char && c < offset_char + value_size);
+        _value = char_to_value[c];
+        return *this;
+    }
+
+    /*!\brief Assign from the numeric phred value.
+     *
+     * \details
+     *
+     * Satisfies the seqan3::quality_concept::assign_phred() requirement via the seqan3::assign_rank() wrapper.
+     *
+     * \par Complexity
+     *
+     * Constant.
+     *
+     * \par Exceptions
+     *
+     * Guaranteed not to throw.
+     */
+    constexpr phred68 & assign_phred(phred_type const p)
+    {
+        // p >= 0 is implicitly always true
+        assert((p >= offset_phred) && (p < value_size + offset_phred));
+        _value = p - offset_phred;
+        return *this;
+    }
+
+    /*!\brief Assign from a the numeric rank value.
+     *
+     * \details
+     *
+     * Satisfies the seqan3::semi_alphabet_concept::assign_rank() requirement via the seqan3::assign_rank() wrapper.
+     *
+     * \par Complexity
+     *
+     * Constant.
+     *
+     * \par Exceptions
+     *
+     * Guaranteed not to throw.
+     */
+    constexpr phred68 & assign_rank(rank_type const p)
+    {
+        // p implicitly always true due to unsignedness of p
+        assert(p < value_size);
+        _value = p;
+        return *this;
+    }
+    //!\}
+
+    /*!\name Comparison operators.
+    * \{
+    */
+    constexpr bool operator==(const phred68 & rhs) const
+    {
+        return this->_value == rhs._value;
+    }
+
+    constexpr bool operator!=(const phred68 & rhs) const
+    {
+        return this->_value != rhs._value;
+    }
+
+    constexpr bool operator<(const phred68 & rhs) const
+    {
+        return this->_value < rhs._value;
+    }
+
+    constexpr bool operator>(const phred68 & rhs) const
+    {
+        return this->_value > rhs._value;
+    }
+
+    constexpr bool operator<=(const phred68 & rhs) const
+    {
+        return this->_value <= rhs._value;
+    }
+
+    constexpr bool operator>=(const phred68 & rhs) const
+    {
+        return this->_value >= rhs._value;
+    }
+    //!\}
+
+    //!\brief The phred score range size for the Solexa standard.
+    static constexpr rank_type value_size{68};
+
```

Comment:
the conversion operators between the quality alphabets are still missing, they can be implemented and tested like for the nucleotide alphabets.